appcues/src/main/java/com/appcues/debugger/ui/CaptureConfirmDialog.kt

```diff
@@ -104,7 +104,7 @@ private fun Header(debuggerViewModel: DebuggerViewModel) {
                     onClick = { debuggerViewModel.closeExpandedView() },
                     role = Role.Button,
                     interactionSource = remember { MutableInteractionSource() },
-                    indication = rememberRipple(bounded = false, 24.dp),
+                    indication = ripple(bounded = false, 24.dp),
```

Comment:
`rememberRipple` --> `ripple` was a required change due to an error level deprecation https://developer.android.com/develop/ui/compose/touch-input/user-interactions/migrate-indication-ripple#migrate-remember-ripple